src/core/modes/walk-mode.ts

```diff
@@ -3,6 +3,20 @@ import { TextEditorDecorationType } from 'vscode';
 import { Mode } from './mode.interface';
 import { objectToCssString } from '../../utils';
 
+type JunbaeLocation = { 'font-size': string } & XOR<{ top: string }, { bottom: string }>;
+type TimerLocation = XOR<{ top: string }, { bottom: string }>;
+
+type Location = {
+  range: vscode.Range;
+  junbaeLocation: JunbaeLocation;
+  timerLocation: TimerLocation;
```

Comment:
Is there a specific reason why the types `JunbaeLocation` and `TimerLocation` are defined separately? It seems like they are only used here and nowhere else.

Reply:
modified